### ISSUE TITLE
Update_project_provisioning breaks code signing on app extensions

### ISSUE THREAD

**Issue by staminajim** [KrauseFx/fastlane#320]
Hi, 

I'm not sure if there's a workaround, but using  `update_project_provisioning` on an app extension seems to break code signing.

This line here:
https://github.com/KrauseFx/fastlane/blob/master/lib/fastlane/actions/update_project_provisioning.rb#L58

forces `CODE_SIGN_RESOURCE_RULES_PATH` to `$(SDKROOT)/ResourceRules.plist` for the given target which causes code signing to fail for app extensions.  That build setting appears to do no harm for the main app target, but makes signing of our app extensions fail during the `ipa` step.  

Clearing `CODE_SIGN_RESOURCE_RULES_PATH` for the app extensions makes the build, sign and archive complete successfully both in `ipa` and Xcode.

 FYI `update_project_provisioning` is successfully updating the profiles and setting `CODE_SIGN_RESOURCE_RULES_PATH`. 

Any ideas on how to solve this?  Could forcing the `CODE_SIGN_RESOURCE_RULES_PATH` build setting be an optional parameter for  `update_project_provisioning` so that we can disable it for app extensions?

This is running on Xcode 6.3.2, fastlane 1.3.1

**Comment by KrauseFx** [comment]
@staminajim thanks for reporting your issue. The [MindNode Setup](https://github.com/fastlane/examples/blob/master/MindNode/Fastfile#L17-L35) is working with App Extensions. Not sure why it's not working for you. 

I'd be okay with adding a flat to disable setting the `ResourceRules`.

**Comment by staminajim** [comment]
Update:

I found out how to get it working.  Whichever version of Xcode that had been unlucky to be used to create the today widget had added a build step which was stripping the contents of the .appex file where the `ResourceRules.plist` lives:
![screen shot 2015-05-29 at 12 36 02 am](https://cloud.githubusercontent.com/assets/1085877/7865884/544f239a-059e-11e5-8459-200258627c30.png)

I deleted that build step and it now works.  I'm unsure why apple decided that was a good idea or if it's necessary for app store submission but it's working for now.

Thanks for your time.
